data/navigation_dataset/examples/objects_in_groups/example_0047.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+  <instruction text="the second and the third chair in the row of chairs aside the robot"/>
```

Comment:
Agreed! I think it is the first time we've seen an example like this. I was not too sure about how exactly to handle the merging of the phrases "the second" and "the third chair".